### PR TITLE
Format the times according to the result

### PR DESCRIPTION
**Pull request by cquiroz**
Signed-off-by: Carlos Quiroz <carlos.quiroz@noirlab.edu>

### CODE REVIEW

**Comment by cquiroz** [comment]
image

File: explore/src/main/scala/explore/itc/ItcColumn.scala [hunk 1]
```diff
@@ -31,7 +31,9 @@ sealed trait ItcResult extends Product with Serializable
 object ItcResult {
   case object SourceTooBright                                     extends ItcResult
   case object Pending                                             extends ItcResult
-  case class Result(exposureTime: FiniteDuration, exposures: Int) extends ItcResult
+  case class Result(exposureTime: FiniteDuration, exposures: Int) extends ItcResult {
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/itc/ItcColumn.scala:34]: When modeling a general representation of a duration of time, we should probably use `java.time.Duration`. It provides arithmetic features and a standarized `toString`, which can be made human presentable with a one-liner (https://stackoverflow.com/a/40487511).

`scala.concurrent.Duration` is meant to be used with `scala.concurrent`.

**Review by rpiaggio — APPROVED** [review]
LGTM. We can switch to `java.time` at another instance.

File: explore/src/main/scala/explore/itc/ItcColumn.scala (hunk 1 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/itc/ItcColumn.scala:34]: I thought about using `Duration` but I feared increasing the js size a lot. But I could try

**Pull request merged**